src/app/beer_garden/db/mongo/util.py

```diff
@@ -164,19 +158,24 @@ def ensure_embedded_command():
     it would be better if we could seamlessly move the existing commands into existing
     Systems.
     """
-    from beer_garden.db.mongo.models import System
+    from .models import Role, System
```

Comment:
Is there a reason why you switched to relative imports? Just curious if you hit an issue that could crop up other places.

Reply:
This was cause by a thing called "Lazy Developer Syndrome", I allowed PyCharm to auto create the import for me. I will switch it back.